dataverse-webapp/src/main/webapp/dataverse_template.xhtml

```diff
@@ -117,6 +117,13 @@
                 bind_bsui_components();
             });
 
+            var originalPrimeFacesAjaxUtilsSend = PrimeFaces.ajax.Request.send;
+            PrimeFaces.ajax.Request.send = function(cfg) {
```

Comment:
Did you try to use `<p:ajaxStatus />`?

Reply:
I've used browser console to check if there are any ajax requests.